docs/infrastructure/adding-hosts-to-fleet.md

```diff
@@ -137,6 +137,8 @@ Next, you'll have to edit the `config.mk` file. You'll find all of the necessary
 
 Note that osqueryd requires a full certificate chain, even for certificates which might be trusted by your keychain. The "Fetch Kolide Certificate" button in the Add New Host screen will attempt to fetch the full chain for you.
 
+If your server certificate is not named kolide.crt, change the certificate name variable in --tls_server_certs to match your server certificate.
```

Comment:
It's not clear to me what program I would pass the `--tls_server_certs` flag. I happen to know that's an `osqueryd` flag, but it is not clear from this excerpt what certificate we're talking about and when I should use this flag.

Reply:
Sorry for the confusion. I mean to say edit the config.mk file line "--tls_server_certs=/etc/osquery/kolide.crt" to match your TLS certificate name instead of the default kolide.crt.